packages/default/test/shared.js

```diff
@@ -88,7 +88,7 @@ export const sharedTestCases = (mode, torusSP, storageLayer) => {
 
       const tb2 = new ThresholdKey({ serviceProvider: customSP, storageLayer: customSL, manualSync: mode });
       await tb2.initialize({ neverInitializeNewKey: true });
-      tb2.inputShareStore(resp1.deviceShare);
+      await tb2.inputShareStoreSafe(resp1.deviceShare);
```

Comment:
are we deprecating inputShareStore and use only inputShareStoreSafe for all usage ?
if so, we can check for if polynomial_id is equal with the tkey's latest polynomial and update it without catchupToLatestShare.

Reply:
hmm, we should make inputShareStore internal or mark internal somehow. With existence of core kit, tkey has more room to be "swiss army knife"